laura/portfolio/src/main/webapp/index.html

```diff
@@ -6,30 +6,35 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  
+    <ul>
+        <li><a href="index.html" class="navbar">Home</a>
+            <a href="mood.html" class="navbar">Mood generator</a></li>
+    </ul>
+
```

Comment:
Navbar should be inside "body".
Also, there is a special tag "<nav>" for elements like this: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

Reply:
done in new commit.